contracts/CO30_Thomas_TanoanSugar.Tanoa/README.md

```diff
@@ -0,0 +1,90 @@
+# Mission Template
```

Comment:
You don't need this file either, just the ones from inside the `mission` folder in the template